### PR TITLE
add hurricane graphic, link

### PR DESCRIPTION
**Pull request by bnfish**

### CODE REVIEW

File: public/css/custom.css [hunk 1]
```diff
@@ -82,6 +82,12 @@ p {
     margin-top: 45px !important;
     float     : none !important;
   }
+  #hurricane-link div {
+    margin-top: 20px !important;
```
Comment (birdage) [inline public/css/custom.css]: @bnfish no importants!!

File: public/css/custom.css [hunk 2]
```diff
@@ -82,6 +82,12 @@ p {
     margin-top: 45px !important;
     float     : none !important;
   }
+  #hurricane-link div {
+    margin-top: 20px !important;
+  }
+  #hurricane-link img {
+    float: none !important;
```
Comment (birdage) [inline public/css/custom.css]: @bnfish no importants!!

Comment (bnfish) [inline public/css/custom.css]: reason?

Comment (birdage) [inline public/css/custom.css]: https://css-tricks.com/when-using-important-is-the-right-choice/

Comment (bnfish) [inline public/css/custom.css]: thank you!

**Pull request merged**